schemas/openapi.yaml

```diff
@@ -106,3 +128,6 @@ components:
               - type: object
                 properties:
                   Right: { $ref: '#/components/schemas/FieldElement' }
+    InclusionProofValidityScope:
+      type: string
+      enum: [ 'sequencer-only', 'fresh-on-chain', 'stable-on-chain' ]
```

Comment:
How about name changes to match blockchain jargon?
* `sequencer-only` -> `pending`
* `fresh-on-chain` -> `submitted` / `minted`
* `stable-on-chain` -> `confirmed`